cmd/waku/server/rest/lightpush_api.yaml

```diff
@@ -0,0 +1,84 @@
+openapi: 3.0.3
+info:
+  title: Waku V2 node REST API
+  version: 1.0.0
+  contact:
+    name: VAC Team
```

Comment:
Also, we should change this to Waku Team right.
cc @NagyZoltanPeter

Reply:
Alright, let us have an issue indicating to use the common REST API spec files once they are placed in their own repo. That issue should address such items then.